sale_order_general_discount_fixed/readme/DESCRIPTION.md

```
@@ -0,0 +1,2 @@
+This module extends the OCA ``sale_fixed_discount`` to allow set discount defined as a fixed monetary amount
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)